src/Controller/Api/FormController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace AppBundle\Controller\Api;
+
+use JMS\Serializer\Serializer;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Validator\ConstraintViolationList;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * @Route("/form")
+ */
+class FormController extends Controller
+{
+    /**
+     * @Route("/validate/{formType}", name="api_form_validate")
+     * @Method("POST")
+     */
+    public function validate(Request $request, Serializer $serializer, string $formType): Response
+    {
+        $form = $this->createForm($formType, null, ['csrf_protection' => false]);
```

Comment:
I am not sure it's a good idea to instantiate something provided by the client.. 
I would prefer a whitelisting approach to avoid any problem.
Also, we could create a mapping of form to avoid passing the whole form type class in the path.
E.g. `/validate/become-adherent` would map to `BecomeAdherentType::class`

Reply:
maybe do it with custom `ParamConvertor` [doc](https://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/converters.html#creating-a-converter)

Reply:
Yes I was thinking about that

Reply:
Not a big fan. It adds an overhead for all requests of all the website for only 2 routes. IMO it's not a good use case for the moment.

Reply:
for now I will make a simple map with key value

Reply:
And if I have the time, create an interface to map all formtype exposed with a static method to get his key

Reply:
There is no need for the formtype to be aware of this thing so the mapping is more than fine.